### DISCUSSION TITLE
Can't switch tabs

### DISCUSSION THREAD

**Discussion by jarrahffe** [LazyVim/LazyVim#332] [Q&A] [answered]
using leader <tab> [ or ] doesnt switch tabs - it just does nothing. Also closing a tab using d also does nothing, how to fix?

**Comment by apinter** [comment]
Same exact issue :(

**Comment by jarrahffe** [comment, marked as answer]
I found the reason. I didn't understand the difference between a tab and a buffer... to switch buffers you can go to the leader, b, then follow the leader guide. a fast way im doing it now is just using [ or ] then b to switch buffer left or right

**Reply by dpetka2001** [reply]
You could also use `<Shift-l>` or `<Shift-h>` to switch between buffers, which i find more convenient than the keys you mentioned. That's just a personal preference.

**Reply by folke** [reply]
Yeah, real tabs are rarely used in Neovim (at least I'm not really using that functionality)
